en/full-node.md

```diff
@@ -220,6 +220,24 @@ have an easy-to-use node.
 
 <div class="toccontent-block boxexpand expanded" markdown="1">
 
+## Initial Block Download(IBD)
+
+[Initial block download](/en/developer-guide#initial-block-download)
+refers to the process where nodes synchronize themselves
+to the network by downloading blocks that are new to them.
+This will happen whenever a new or offline node starts up.
+
+If you are trying to set up a new node following the instructions below, you will go
+through the IBD process at the first run, and it may take a considerable amount of time since a new
+node has to download the entire blockchain(which is more than 100 gigabytes by the time this paragraph is written).
```

Comment:
1. There should be a space between "blockchain" and "(which"; same for the parenthesis is the subsequent paragraph.

2. For the size of the block chain, you can use the variable `{{site.text.chain_gb}}`, which is used elsewhere in this document if you want to see an example.  This allows someone to update the variable definition in `_config.yml` periodically to reflect the current chain size.  If you want to do it in this PR, this variable can probably be updated now as it says 145 GB but the current size is,

```text
$ du -sh .bitcoin/blocks/
183G    .bitcoin/blocks/
```